pom.xml

```diff
@@ -585,7 +585,15 @@
                         <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
                         <artifactSet>
                             <excludes>
-                                <exclude>org.slf4j:*</exclude>
+                                <!-- not excluded: jcl-over-slf4j(commons-logging to slf4j bridge) -->
+                                <exclude>org.slf4j:slf4j-api</exclude>
+                                <exclude>org.slf4j:slf4j-jdk14</exclude>
+                                <exclude>org.slf4j:slf4j-jcl</exclude>
+                                <exclude>org.slf4j:slf4j-nop</exclude>
+                                <exclude>org.slf4j:slf4j-simple</exclude>
+                                <exclude>org.slf4j:slf4j-reload4j</exclude>
+                                <exclude>org.slf4j:slf4j-log4j12</exclude>
+                                <exclude>org.slf4j:log4j-over-slf4j</exclude>
```

Comment:
revert

Changing here has no effect

Reply:
Using your method to test whether the distribution will exclude<jul-to-slf4j>, as shown in the screenshot below.
<img width="966" alt="image" src="https://user-images.githubusercontent.com/40714172/218620371-0795324c-cf04-40c5-9e43-ec4789dd54ae.png">
My method is effective, as shown in the screenshot below.
<img width="916" alt="image" src="https://user-images.githubusercontent.com/40714172/218620475-9a4c23e7-c11b-4386-83b2-cf542b8f4795.png">

Reply:
> 

ok, Is this problem to be handled by users? or are there other solutions?

Reply:
> 1. enable this dependency
>    https://github.com/apache/incubator-seatunnel/blob/dev/pom.xml#L185
> 2. add dependency to this location
>    https://github.com/apache/incubator-seatunnel/blob/dev/pom.xml#L487
> 
> ```
>             <dependency>
>                 <groupId>org.slf4j</groupId>
>                 <artifactId>jul-to-slf4j</artifactId>
>             </dependency>
> ```
> 
> 3. add pulsar connector e2e
>    https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e

Here are my test results

```shell
./mvnw -T 1C clean package -pl seatunnel-core/seatunnel-flink-starter/seatunnel-flink-13-starter -am -Dmaven.test.skip=true -Pseatunnel
```

<img width="957" alt="image" src="https://user-images.githubusercontent.com/14371345/218926205-67e73e8d-7ca5-45b7-8ba9-463968fb3e16.png">

Reply:
ok，thanks.